include/uxr/client/core/session/session.h

```diff
@@ -28,10 +28,25 @@ extern "C"
 struct uxrSession;
 struct uxrCommunication;
 
-typedef void (*uxrOnStatusFunc) (struct uxrSession* session, uxrObjectId object_id, uint16_t request_id,
-                             uint8_t status, void* args);
-typedef void (*uxrOnTopicFunc) (struct uxrSession* session, uxrObjectId object_id, uint16_t request_id,
-                             uxrStreamId stream_id, struct ucdrBuffer* ub, void* args);
+typedef void (*uxrOnStatusFunc) (struct uxrSession* session,
+                                 uxrObjectId object_id,
+                                 uint16_t request_id,
+                                 uint8_t status,
+                                 void* args);
+
+typedef void (*uxrOnTopicFunc) (struct uxrSession* session,
+                                uxrObjectId object_id,
+                                uint16_t request_id,
+                                uxrStreamId stream_id,
+                                struct ucdrBuffer* ub,
+                                void* args);
+
+typedef void (*uxrOnTimeFunc) (struct uxrSession* session,
```

Comment:
In what cases is this callback needed for the user?

Reply:
The user could be interested in implement her/his own time synchronization algorithm.
For example, she/he could perform a statistical analysis.

Reply:
What happened (with the stored memory of the callback) in the common case where the user does not want to implement its own sync algorithm? Should these attributes only added to the `Session` structure by a defined flag?

Reply:
Yes, we can add a CMake flag to enable/disable this feature. Do you think it is better to add another flag just for few bytes on the stack? Please @BorjaOuterelo, add your point of view.

Reply:
@julianbermudez, @lemunozm  Let's go with the callback for the moment.